src/styles/custom.less

```diff
@@ -55,3 +55,10 @@
 @layout-trigger-height : 48px;
 @layout-zero-trigger-width : 36px;
 @layout-zero-trigger-height : 42px;
+
+// Comments
+@comment-text-left-margin: 12px;
```

Comment:
This is great, thanks for providing more context on what the values actually mean. @bonustrack what do you think of this fix? It seems to be working for me.